tests/app/filter.test.ts

```diff
@@ -63,7 +63,21 @@ const selectIfSet = async (
   values?: string[]
 ): Promise<void> => {
   if (values !== undefined) {
-    await page.locator(selector).selectOption(values);
+    // Reset filters by unchecking all
+    const checkboxes = await page.$$(
+      `input[type="checkbox"][name="${selector}"]`
+    );
+    /* eslint-disable */
+    for (const checkbox of checkboxes) {
+      await checkbox.uncheck();
+    }
+
+    for (const value of values) {
+      const label = await page.$(`label:has-text("${value}")`);
+      const checkbox = await label.$('input[type="checkbox"]');
```

Comment:
Is it possible to reuse the variable `checkboxes`? (This question isn't blocking)

Reply:
I don't think so. Right now, the checkboxes are inside the label, so to be able to find the label, we'd have to find the parent element of the checkbox, which I don't think is possible?

There are probably better ways to do this, like adding `value="option name"` for each checkbox.